csaf_2.0/submit/README.md

```diff
@@ -0,0 +1,84 @@
+# Submission request to advance CSAF v2.0 to an International Standard
+
+Date: January 28, 2023
+
+>*The following text parts - marked up in a gray, cursive,, and smaller font - have been copied directly from*
+>*the requirements stated in the relevant section of the current OASIS Liaison Policies page at:*
+>                                          https://www.oasis-open.org/policies-guidelines/liaison/#submitwork
+
+*Any submission request delivered to the OASIS President under this policy must be in writing, and must include the following:*
+
+a. *The name(s) of the submission requester(s), that is, the TC, the Member Section, or the OASIS*
+   *Organizational Members that support the submission request as described in section 1(d).*
+
+OASIS Common Security Advisory Framework (CSAF) TC
+
+b. *The name of the intended receiving standards organization. The request may also suggest the committee*
+   *or group in that organization which should process that submission.*
+
+ISO/IEC\ JTC\ 1 Secretariat (suggesting “SC\ 27 Information security, cybersecurity and privacy
+protection” as the published scope at https://www.iso.org/committee/45306.html indicates)
+
+c. *The intended status or outcome that the request seeks from the receiving organization’s process; and a*
+   *short description of the receiving organization’s approval process, including estimated time required,*
+   *stages of approval and who votes at each stage.*
+
+Intended status: Advance CSAF\ v2.0 OASIS to an ISO/IEC International Standard
```

Comment:
Added more precision by amending OASIS as OASIS Standard in a later commit [6544f2ae...](https://github.com/oasis-tcs/csaf/pull/603/commits/6544f2ae5610c04f77b18ca4fbbb0301b1c03333)